### DISCUSSION TITLE
Confuse about how to do challenge 4.

### DISCUSSION THREAD

**Discussion by SCPassion** [Cyfrin/foundry-full-course-cu#671] [Q&A] [answered]
This might be a silly question. Should I download all the contracts from Patrick on Etherscan and deploy the contract on Remix or similar platforms?

What is the best way to execute the function?
```
    function getPrice() private view returns (uint256) {
        (, int256 price,,,) = AggregatorV3Interface(i_priceFeed).latestRoundData();
        return uint256(price);
    }
```

Could somebody give some guidelines on how to interact with the contract?
Thanks a lot.

**Comment by 0xusmanf** [comment, marked as answer]
You won't be able to interact with the function directly as it is a private function. Try using Remix to mitigate the function.
